containers/groovy-examples/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.4"
+
+services:
+  grpc-api:
+    image: ghcr.io/deephaven/grpc-api:${VERSION:-latest}
+    expose:
+      - '8080'
+    volumes:
+      - ./docker/core/data:/data
+      - api-cache:/cache
+    environment:
+      - JAVA_TOOL_OPTIONS=-Xmx4g -Dio.deephaven.console.type=groovy
+
+
+  web:
+    image: ghcr.io/deephaven/web:${VERSION:-latest}
+    expose:
+      - '80'
+    volumes:
+      - data:/data
+      - web-tmp:/tmp
+
+  grpc-proxy:
+    image: ghcr.io/deephaven/grpc-proxy:${VERSION:-latest}
+    environment:
+      - BACKEND_ADDR=grpc-api:8080
+    depends_on:
+      - grpc-api
+    expose:
+      - '8080'
+
+  envoy:
+    image: ghcr.io/deephaven/envoy:${VERSION:-latest}
+    depends_on:
+      - web
+      - grpc-proxy
+      - grpc-api
+    ports:
+      - "${PORT:-10000}:10000"
+
+  examples:
+    image: ghcr.io/deephaven/examples
+    volumes:
+      - ./docker/core/data:/data
```

Comment:
The `./docker/core/data` path seems funny for this use case.  Kind of Rube Goldberg.  Should we be using `./data` instead?